runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -4087,6 +4101,10 @@ TR::Register *J9::Power::TreeEvaluator::VMinstanceOfEvaluator(TR::Node *node, TR
    if (objectClassReg)
       srm->reclaimScratchRegister(objectClassReg);
 
+   if (generateBranchToOOL) {
```

Comment:
we are kind of forced by these evaluator framework generating this extra unconditional/taken branch.  have you considered how we can improve that?

Reply:
That is true. Due to the way the code already exists, added this condition after while loop. In this specific case, it does not enter the while loop, so it is valid to generate branch earlier. Will do the changes and verify and push change.